### ISSUE TITLE
Add a setting option to specify the module directory

### ISSUE THREAD

**Issue by Luthaf** [Luthaf/linter-gfortran#6]
As proposed in #5, it would be nice if one could set the module directory (`-J` gfortran option) in the package settings.

**Comment by AndreHorst** [comment]
...maybe even for each project individually, but I don't know how this could work. Just an idea!

**Comment by Luthaf** [comment]
Yes, that could be nice. We could follow the linter-clang/linter-gcc approach here and have a per-project configuration file (`.clang_complete` and a JSON file for linter-gcc)

**Comment by Arcanemagus** [comment]
For per-project settings in other linters we defer people to [project-manager](https://atom.io/packages/project-manager) (at least until Atom finally brings in support natively for per-project settings).

This would of course still require a setting for this to be added first 😉.

**Comment by RCarvalho13** [comment]
I recently switched from Sublime Text to Atom (best decision ever) and, even though the linter-gfortran works great for most cases, I'm still getting some troubles when dealing with modules. The compiler cannot find the associated .mod files (naturally) and the only solution that is working for me is to just open all the module files, temporarily solving the issue. Anyone has any suggestions on how to solve this?

**Comment by AndreHorst** [comment]
is [#5](https://github.com/AtomLinter/linter-gfortran/issues/5) not working? When I tried it, the modules were found!

**Comment by RCarvalho13** [comment]
Unfortunately no, I already tried the following settings and it just stops working and doesn't detect any errors. I'm using gfortran v7.2, can this be a problem?
![image](https://user-images.githubusercontent.com/24481768/28878964-bcd74456-7798-11e7-8368-b0a750515b38.png)

**Comment by AndreHorst** [comment]
hmm... I tried it with fortran 4.9.2 and it works. In [this](https://stackoverflow.com/questions/8855896/specify-directory-where-gfortran-should-look-for-modules) answer, the J-flag is the first one. Maybe this works?

**Comment by RCarvalho13** [comment]
That thread includes a very useful information with regard to the use of the "J" flag. :+1: 

Meanwhile I've tested the gfortran-5 and the linter still doesn't work. Although I think I now the source of the problem: I have some modules using other modules and those dependencies are not provided with the -J flag.
Any idea on how to solve this?

**Comment by AndreHorst** [comment]
hm, you could link all .mod files into a seperate directory where only linter checks for the modfiles. This directory will be ignored during you compiling routine but be visible for linter itself. I would be interested, if this works! :-)
